src/components/CustomerLogos.tsx

```diff
@@ -1,80 +1,121 @@
 import React, { FunctionComponent } from 'react'
 
-interface Logos {
+import Link from 'next/link'
+
+interface ImageWrapper {
+    isLink: boolean
+    logo: Logo
+    children: React.ReactNode
+}
+
+interface Logo {
     name: string
     src: string
+    link: string
+}
+
+interface CustomerLogos {
+    isLink?: boolean
```

Comment:
That articles "good" examples use `is` and `has`.  I agree these variables read better without them. Using adjectives instead of verbs for variables sounds good to me ⚡

Reply:
Ah gotcha! Woah so many opinions there 😮 -- yea I agree with your example 🤜🏻 🤛🏻  I'll update any naming I come across to our new convention 😄